config/application.rb

```diff
@@ -31,5 +31,6 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+    config.time_zone = 'Eastern Time (US & Canada)'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.